Lib/test/test_dataclasses/__init__.py

```diff
@@ -4869,5 +4869,125 @@ class A:
         self.assertEqual(fs[0].name, 'x')
 
 
+class TestZeroArgumentSuperWithSlots(unittest.TestCase):
+    def test_zero_argument_super(self):
+        @dataclass(slots=True)
+        class A:
+            def foo(self):
+                super()
+
+        A().foo()
+
+    def test_zero_argument_super_with_old_property(self):
```

Comment:
This test name is a bit odd, since there's no use of `super()` at all in the test.
```suggestion
    def test_dunder_class_with_old_property(self):
```